Add write batch

trK54Ylmz opened the pull request:
I have added write batch and updated unit tests.

Sorry for diff from previous PR.

xyb commented:
@trK54Ylmz I have merged #6, please rebase to the latest master branch. Thank you!

trK54Ylmz commented:
I have removed repository today, but when I see the comment forked again and updated changes but PR does not sync the updates. is it okay, if I close this one and create new PR @xyb?

Also I started to get the same exact error on my Mac [5054861329](https://github.com/xyb/rocksdb3/runs/5054861329?check_suite_focus=true#step:10:31),

```
Caused by:
  process didn't exit successfully: `/Users/runner/work/rocksdb3/rocksdb3/target/release/build/librocksdb-sys-2b31[40](https://github.com/xyb/rocksdb3/runs/5054861329?check_suite_focus=true#step:10:40)e0065ce4f7/build-script-build` (exit status: 101)
  --- stderr
  error: header 'rocksdb/include/rocksdb/c.h' does not exist.
  thread 'main' panicked at 'unable to generate rocksdb bindings: ()', /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/librocksdb-sys-6.28.2/build.rs:44:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Traceback (most recent call last):
  File "/Users/runner/work/rocksdb3/rocksdb3/tests/build.py", line 16, in <module>
    subprocess.run(["cargo", "build", "--release"], check=True, cwd=str(ROOT))
  File "/Users/runner/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['cargo', 'build', '--release']' returned non-zero exit status 101.
Error: Process completed with exit code 1.
```

xyb commented:
@trK54Ylmz sure, please do what you find handy.
